.circleci/export-e2e-test-user-vars.sh

```diff
@@ -3,36 +3,46 @@ set -xv
 
 ENV=${1:-test}
 
+function getProp() {
+  local fileName=$1
+  local key=$2
+  grep "${key}" "${fileName}"|cut -d'=' -f2
+}
+
 for arg in "$@"
 do
   case "$arg" in
     "perf")
-       echo "export READER_USER=$PUPPETEER_READER_PERF" >> "$BASH_ENV"
-       echo "export WRITER_USER=$PUPPETEER_WRITER_PERF" >> "$BASH_ENV"
-       echo "export COLLABORATOR_USER=$PUPPETEER_COLLABORATOR_PERF" >> "$BASH_ENV"
-       echo "export USER_NAME=$PUPPETEER_USER_PERF" >> "$BASH_ENV"
-       echo "export ACCESS_TEST_USER=$PUPPETEER_ACCESS_PERF" >> "$BASH_ENV"
+       echo "export DOTENV_CONFIG_PATH=.env.circleci.perf" >> "$BASH_ENV"
```

Comment:
Could this entire script be replaced by:

```
cp ".env.circleci.${WORKBENCH_ENV}" .env
```
?

Is there a reason to actually export these variables anymore?

Reply:
I think it's needed. I'll run more tests.